deku-c/wasm-vm-ocaml/operation.ml

```diff
@@ -0,0 +1,17 @@
+open Deku_ledger
+open Deku_concepts
+
+type t = {
+  operation : Operations.t;
```

Comment:
I don't see why there's an Operation and an Operations module and how operations could more than one thing, the names should probably change in order to settle the confusion.